### PR TITLE
Bug 1080148 - Default contact picture should contain the first letter of the contact name

### PR DESCRIPTION
**Pull request by kumarrishav**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4a3b91ab13ad7675824ba8234b2656a93788dd63

File: apps/sms/views/inbox/js/inbox.js [hunk 1]
```diff
@@ -217,6 +217,9 @@ var InboxView = {
         ].join(', ');
       } else {
         picture.firstElementChild.style.backgroundImage = null;
+        if(contact.title) {
+          picture.firstElementChild.textContent = title.textContent[0];
```
Comment (julienw) [inline apps/sms/views/inbox/js/inbox.js]: why do'nt you use contact.title[0] ?

File: apps/sms/views/inbox/js/inbox.js [hunk 2]
```diff
@@ -217,6 +217,9 @@ var InboxView = {
         ].join(', ');
       } else {
         picture.firstElementChild.style.backgroundImage = null;
+        if(contact.title) {
```
Comment (julienw) [inline apps/sms/views/inbox/js/inbox.js]: nit: space after "if"

File: apps/sms/views/inbox/js/inbox.js (hunk 1 above)
Comment (kumarrishav) [inline apps/sms/views/inbox/js/inbox.js]: hmm..yeah. that can also be used.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f41695354caa7b242cbc7ab795298d8ea44351b4

**Pull request closed without merging**